lang/tr.xml

```diff
@@ -367,7 +364,6 @@ Devam etmek istediğinizden emin misiniz?</user_delete_warn>
 		<change_avatar>Avatar Değiştir</change_avatar>
 		<reset_avatar>Avatar Sıfırla</reset_avatar>
 		<name>Ad</name>
-		<user>Kullanıcı</user>
```

Comment:
Revert

Reply:
here is display "User1"

Reply:
Ah 🤔 ok

Reply:
Is ps4 that ? Is on what device ?

Reply:
yes, is on ps4, i used remote play on my vita

Reply:
ah is vita lol

Reply:
you can show me same screen in french ? i am curius for one things here

Reply:
> ah is vita lol

yes, Mr. Zangetsu XD

Reply:
> you can show me same screen in french ? i am curius for one things here

sure, give me a second